src/main/java/com/google/api/codegen/grpcmetadatagen/GrpcMetadataGenerator.java

```diff
@@ -135,34 +64,17 @@ protected void process() throws Exception {
       }
       return;
     }
-
-    writeToFile(generateDocs(model));
-  }
-
-  private void writeToFile(Map<String, Doc> docs) throws Exception {
-    ToolUtil.writeFiles(docs, options.get(OUTPUT_DIR));
-  }
-
-  private static String outputPath(String templateFileName) {
-    String baseName = Paths.get(templateFileName).getFileName().toString();
-    int extensionIndex = baseName.lastIndexOf(".");
-    return baseName.substring(0, extensionIndex);
-  }
-
-  private static List<String> getSnippets(TargetLanguage language) {
-    return getForLanguage(SNIPPETS, language);
-  }
-
-  private static GrpcPackageCopier getCopier(TargetLanguage language) {
-    return getForLanguage(COPIERS, language);
+    ToolUtil.writeFiles(generate(model), options.get(OUTPUT_DIR));
```

Comment:
put `generate(model)` on its own line - it is the most important step, but it's kind of hidden here.